### DISCUSSION TITLE
Application Container Logs - Best Practices?

### DISCUSSION THREAD

**Discussion by GLStephen** [community/community#5612] [Codespaces]
What is the best practice for accessing application logs in a Codespaces container? Our default config for containers is to log to stderr, but I don't see a way to access those logs in codespaces. What's the best practice here?

**Comment by tanmayeekamath** [comment]
Hey @GLStephen, here's a doc on how to view and export codespace logs: https://docs.github.com/en/codespaces/codespaces-reference/using-the-command-palette-in-codespaces#codespaces-logs

Does this help? :)
